### PR TITLE
Adapting the FindRecipeScreen

### PR DESCRIPTION
**Pull request by mche3kek**
Adapting the FindRecipe and Camera screens to delete the previous outlined buttons and adapt the interface for a more user-friendly experience.

We use floating buttons for validating the ingredients inputted and accessing the camera, and we adapted the camera screen to include the option of choosing a picture along with deleting the previous Gallery screen.

### CODE REVIEW

**Review by ohamz — CHANGES_REQUESTED** [review]
Overall, I think the UI of the screen is a lot more intuitive (it will become more intuitive with a better UI of ingredients input that we will implement in another PR).
Also for the Camera screen, I find it better as a user to click on the barcode button (for example) directly, instead of first taking a photo manually.

**Comment by mche3kek** [comment]
> Overall, I think the UI of the screen is a lot more intuitive (it will become more intuitive with a better UI of ingredients input that we will implement in another PR). Also for the Camera screen, I find it better as a user to click on the barcode button (for example) directly, instead of first taking a photo manually.

In fact, I have a PR next week to adapt the Camera Screen, we will only have the camera or gallery picture first, then after we display the picture taken or chosen and we will have the two buttons for barcode scanning or text recognition for a more user-friendly approach and to be more intuitive

File: app/src/main/java/com/android/feedme/ui/find/FindRecipeScreen.kt [hunk 1]
```diff
@@ -73,67 +94,67 @@ fun FindRecipeScreen(navigationActions: NavigationActions, inputViewModel: Input
...
+              //                        style =
+              //                            TextStyle(
+              //                                fontSize = 16.sp,
+              //                                lineHeight = 20.sp,
+              //                                fontWeight = FontWeight(700),
+              //                                color = FindRecipeIcons,
+              //                                textAlign = TextAlign.Center,
+              //                                letterSpacing = 0.25.sp,
+              //                            ))
+              //                  }
```
[the remark this answers is not in the document]
Comment (mche3kek) [reply app/src/main/java/com/android/feedme/ui/find/FindRecipeScreen.kt:157]: I will on my next commit, thanks for the suggestion !

**Comment by ohamz** [comment]
> In fact, I have a PR next week to adapt the Camera Screen, we will only have the camera or gallery picture first, then after we display the picture taken or chosen and we will have the two buttons for barcode scanning or text recognition for a more user-friendly approach and to be more intuitive

Great ! That sounds like a good idea.

**Comment by mche3kek** [comment]
> ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=feedme-swent-epfl_feedme-android&pullRequest=185) **Quality Gate failed**
> 
> Failed conditions  [67.86% Line Coverage on New Code](https://sonarcloud.io/dashboard?id=feedme-swent-epfl_feedme-android&pullRequest=185) (required ≥ 80%)
> 
> [See analysis details on SonarCloud](https://sonarcloud.io/dashboard?id=feedme-swent-epfl_feedme-android&pullRequest=185)

The uncovered lines are related to the gallery launcher, which we cannot test without injecting images to our emulator. therefore it is fine to merge it as is

**Review by ohamz — APPROVED** [review]
I agree that testing the couple of lines in the CameraViewModel can be ignored, we tried in the past for several hours but we failed ...

Anyway, this PR is approved !

**Pull request merged**